### PR TITLE
Encoding utf-8 fix

### PR DESCRIPTION
**Pull request by madlew**
Since we have polish character in the rspec-hive.gemset gemfile, we couldn't remove encoding comment (as rubocop suggested). Without this comment bundle install mostly work but not in all cases.

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -41,7 +42,8 @@ GEM
     guard-rubocop (1.2.0)
       guard (~> 2.0)
       rubocop (~> 0.20)
-    i18n (0.8.6)
+    i18n (0.9.0)
```
Comment (jekuta) [inline Gemfile.lock]: Why was this updated? Not saying it is bad, just why is it included here?

Comment (madlew) [reply Gemfile.lock]: so it happened automatically with this dependency when I run bundle install

Comment (jekuta) [reply Gemfile.lock]: Hm, that sounds weird

Comment (madlew) [reply Gemfile.lock]: why? I do believe bundle resolves all the dependencies (somehow) and in that case it's rather out of our controll. Either we believe it works ok or not. On the other hand I do agree I'd expect such an update would happen rather when we call bundle update not bundle install

Comment (jekuta) [reply Gemfile.lock]: Yeah, in normal flow this should never happen. The whole point of Gemfile.lock is that when you download the repo and run bundle install, you get exactly what is described in Gemfile.lock and it is not going to change just by running `bundle install`

Comment (madlew) [reply Gemfile.lock]: so as you noticed in a different comment, the gem version was changed to 0.5.0 without calling bundle install, then I updated gemfile.lock which is part of this PR (https://github.com/u2i/rspec-hive/pull/34/commits/991ef11fd6e5d4200e31c3827167fdd3d21e23d5), then decided to bump to 0.5.1 due to fix and updated gemfile.lock (via bundle install) again

Comment (madlew) [reply Gemfile.lock]: i18n is not constrained on our end in any way so maybe that's why bundle decided it can be bumped since there's a newer version

Comment (mkrawc) [reply Gemfile.lock]: The tricky part of Gemfile.lock is that it _should_not_ be under version control in gems. Gems are meant to be universal to some degree and should work with all versions that are allowed in gemspec.

Comment (jekuta) [reply Gemfile.lock]: Yeah that is correct

**Pull request merged**